src/routes/+layout.ts

```diff
@@ -19,11 +20,12 @@ export const load: LayoutLoad = async ({ url, fetch }) => {
         throw error(401, 'Unauthenticated');
     }
 
-    const [languages, parentResources, resourceContentStatuses, currentUser] = await Promise.all([
+    const [languages, parentResources, resourceContentStatuses, currentUser, helpDocuments] = await Promise.all([
         getFromApi<Language[]>('/languages', fetch),
         getFromApi<ParentResource[]>('/resources/parent-resources', fetch),
         getFromApi<ResourceContentStatus[]>('/resources/content/statuses', fetch),
         getFromApi<CurrentUser>('/users/self', fetch),
+        getFromApi<HelpDocumentResponse>('/help', fetch),
```

Comment:
What do you think about doing this fetch within the component itself using a loading indicator there? It shouldn't be a heavy call by any means, but it's still one extra call happening every page load that isn't really needed until they open the modal.

Reply:
Since it is in the `layout` file is only fetched once, just as the `/self`. So, subsequent navigation in the app does not make a call to the `/help`.

However, I do like the idea of only calling the endpoint if it's needed and saving as much overhead on an initial load as possible.

Reply:
Kyle is right. This can get called on every page load like this. This needs to be moved to the page/component itself.